egs/tedlium/s5/local/chain/README

```diff
@@ -0,0 +1,31 @@
+These are instruction to reproduce the TEDLIUM models described in
+"Purely sequence-trained neural networks for ASR based on lattice-free
+MMI", by Povey et al.
+
+First run:
+
+./run.sh
+
+until the end of stage 7. (local/nnet/run_dnn.sh can be skipped.)
+
+Then run:
+
+local/chain/run_tdnn.sh
+
+to see results for a generic chain model (these were not the results
+shown in the paper).
+
+Only after that, you can run:
+
+local/chain/run_tdnn_more_ce.sh
```

Comment:
Yes, I don't think we need to have two scripts differing so slightly.   If
you want to record the results you could state e.g. in the top of the file,
that when you ran it with standard cross-entropy smoothing value of
cross_entropy_smoothing=0.1 (or whatever it is), you got such-and-such
results.
Dan


On Sat, Apr 2, 2016 at 5:14 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/tedlium/s5/local/chain/README
> <https://github.com/kaldi-asr/kaldi/pull/644#discussion_r58300002>:
>
> > +First run:
> > +
> > +./run.sh
> > +
> > +until the end of stage 7. (local/nnet/run_dnn.sh can be skipped.)
> > +
> > +Then run:
> > +
> > +local/chain/run_tdnn.sh
> > +
> > +to see results for a generic chain model (these were not the results
> > +shown in the paper).
> > +
> > +Only after that, you can run:
> > +
> > +local/chain/run_tdnn_more_ce.sh
>
> This script should be renamed as local/nnet3/run_tdnn.sh
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/644/files/82caa06f43641485acb81b204e02eea21e5bbbeb#r58300002>
>